server/utils/ssr.tsx

```diff
@@ -220,14 +220,21 @@ export const generateMetaComponents = (metaProps: MetaProps) => {
 	}
 
 	if (attributions) {
-		const authors = attributions.sort(sortAttributions).filter((item) => {
-			return item.isAuthor && !item.roles;
-		});
+		const authors: Attribution[] = [];
+		const contributors: Attribution[] = [];
 
 		const getPrimaryRole = (contributor: Attribution) => contributor.roles?.[0];
```

Comment:
@gabestein I wanted to confirm two more things about how this _should_ work:

1. We're only considering the first role in the list
2. The `<meta name="dc.author">` tags should follow the same rules as the `<meta name="citation_author">` ones (create one for each author whose primary role is not a contributor role).

Reply:
👍 to both!

One note for posterity: it does look like [Dublin Core](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/) has shifted its taxonomy to use `dc.creator` rather than `dc.author`, but I'm not sure how well supported this is by the tools consuming this metadata, so we should probably stick with author for now. At some point we may want to make the switch.